numba/tests/npyufunc/test_cuda_gufunc.py

```diff
@@ -0,0 +1,190 @@
+from __future__ import print_function, absolute_import
```

Comment:
I'm thinking about something... if these are all CUDA tests, shouldn't they be in `numba/cuda/tests/cudapy/`?